src/server/db/inmemory.js

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Can you branch this file from db.js like you did for mongo.js?

Reply:
I'm not sure I understood what you want here. Can you elaborate a little bit more please?

Reply:
When I look at the diff for mongo.js, I see that it was renamed from db.js (as opposed to a delete + add). Looks like this is git automatically detecting a rename. Never mind, I don't think there is a git way to do it for another file.

Reply:
Oh ok.. Now I understand. 
Yes it was a automatic detection. I deleted files after I created all and I suppose git chose mongo file to be set as renamed. Probably because I created that one first or last.